models/experimental/grok/tests/test_grok_decoder.py

```diff
@@ -0,0 +1,124 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+import os
+import torch
+import pytest
+from loguru import logger
+
+# Set Grok flags for CI, if CI environment is setup
+if os.getenv("CI") == "true":
+    os.environ["GROK_CKPT_DIR"] = "/mnt/MLPerf/tt_dnn-models/Grok/Grok-1/"
+    os.environ["GROK_TOKENIZER_PATH"] = "/mnt/MLPerf/tt_dnn-models/Grok/Grok-1/"
+    os.environ["GROK_CACHE_PATH"] = "/mnt/MLPerf/tt_dnn-models/Grok/Grok-1/"
+    os.environ["TT_METAL_ASYNC_DEVICE_QUEUE"] = "1"
+    os.environ["WH_ARCH_YAML"] = "wormhole_b0_80_arch_eth_dispatch.yaml"
+
+import ttnn
+from models.experimental.grok.tt.grok_common import prepare_inputs_ttnn, prepare_rotation_mat_ttnn
+from models.experimental.grok.tt.grok_decoder import TtTransformerBlock
+from models.experimental.grok.reference.model import DecoderLayer
+from models.experimental.grok.tt.model_config import TtModelArgs
+from models.utility_functions import comp_pcc, comp_allclose
+from ttnn import ReplicateTensorToMesh, ConcatMeshToTensor
+
+
+@pytest.mark.timeout(500 * 8)
```

Comment:
Can we remove these timeouts?
We use xdist for timeouts which handles inadvertent hangs/crashes for pytests.

Follows the format such as this ripped from run_t3000_demo_tests.sh

Reply:
Sure I can move them to the script level